source/index.html.md

```diff
@@ -183,6 +183,35 @@ This endpoint finds  customer record by its id
 
 This endpoint loads customers by csv files
 
+>**Example body for uploading a csv file containing customer records**
+
```

Comment:
What you have here is not the body. It's the response to the operation. The body is the CSV file. So label it appropriately